server/cache.go

```diff
@@ -73,16 +67,16 @@ func loadClusterInfo(id core.IDAllocator, kv *core.KV) (*clusterInfo, error) {
 	}
 
 	start := time.Now()
-	if err := kv.LoadStores(c.stores, kvRangeLimit); err != nil {
+	if err := kv.LoadStores(c.Stores, kvRangeLimit); err != nil {
 		return nil, errors.Trace(err)
 	}
-	log.Infof("load %v stores cost %v", c.stores.GetStoreCount(), time.Since(start))
+	log.Infof("load %v Stores cost %v", c.Stores.GetStoreCount(), time.Since(start))
```

Comment:
using stores and regions in the log and comment is ok, no need to change them.

Reply:
done